lib/entity_cache/storage/persistent.rb

```diff
@@ -2,10 +2,11 @@ class EntityCache
   module Storage
     module Persistent
       extend Configure::Macro
-      configure :persistent_store
 
       def self.included(cls)
         cls.class_exec do
+          configure :persistent_store
```

Comment:
I would not mind it one bit. The only surprising thing is that it defines a `.build` factory method but does not include the word "build" in the method name.

This is a pattern that is so pervasive, I think, that we can tolerate a bit of a learning curve.